.github/workflows/coverage-windows.yml

```diff
@@ -0,0 +1,51 @@
+name: coverage-windows
+
+on:
+  pull_request:
+    paths-ignore:
+      - 'doc/**'
+      - 'deps/**'
+      - 'benchmark/**'
+      - 'tools/**'
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - 'doc/**'
```

Comment:
```suggestion
      - '.github/**'
      - 'doc/**'
```

Reply:
But we would want to run this workflow if the workflow itself (`.github/workflows/coverage-windows.yml`) was modified.

Reply:
@richardlau @aduh95 would you be alright with us just running on changes to .GitHub, for the time being.

I'd rather this than get overly fancy with the glob.

Reply:
Sure, works for me 👍